src/components/application_manager/test/help_prompt_manager_test.cc

```diff
@@ -180,7 +180,8 @@ void HelpPromptManagerTest::SetUp() {
       app_mngr_,
       HmiState::STATE_ID_REGULAR);
 
-  std::string path = file_system::CreateDirectory("storage");
+  std::string path("storage");
+  file_system::CreateDirectory(path);
```

Comment:
@SKobziev You have changed return value to `bool` - so why did you ignore it?

Reply:
@AByzhynar It is corrected. Could you please check it?